main.go

```diff
@@ -26,6 +26,7 @@ type config struct {
 	DepWarnning  string
 	Script       []string `yaml:"script"`
 	Build        []string `yaml:"build"`
+	Runners      []string `yaml:"runners"`
```

Comment:
Right, I think simply `run` describes what will happen to them. If we happen to add functionality down the line that allows these things to be ran sequentially we would have the option of adding `async` blocks etc...